creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -648,6 +654,23 @@ class SnapbackSM {
         throw new Error('processStateMachineOperation():retrieveClockStatusesForSecondaryUsersFromNodes() Error')
       }
 
+      // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
+      try {
+        const endpointToSpIdMap = await this.createEndpointToSpIdMap()
```

Comment:
i think this is a bit different than what my understanding of our discussion was. but either way, i think it may be cleaner to encapsulate this try-catch logic in the `createENdpontToSpIDMap()` function instead, ie the function interlaly has a try-catch and overrides the class var if successful, and always returns the latest value of the class var, the caller never needs to know about that

Reply:
this try-catch block is a bit confusing, with the conditional flipping of `snapbackReconfigEnabled`.

i thought if the endpoint-spid refresh ever failed we'd just use the previously fetched value?

Reply:
addressed ~